test/run_pass/test_iotjs_error_stack.js

```diff
@@ -0,0 +1,19 @@
+var assert = require('assert');
+
+var error = getError();
+assert(typeof error.stack === 'string');
+assert(error.stack.match(/^    at getError/));
```

Comment:
A little suggestion, could we changing this match rule to simple `/at GetError/`? That don't make me in trouble whenever we tweak the spaces :)